src/components/sidebar/CallTreeSidebar.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import * as React from 'react';
+import explicitConnect from '../../utils/connect';
+import { selectedThreadSelectors } from '../../reducers/profile-view';
+import { getFunctionName } from '../../profile-logic/function-info';
+
+import type { ExplicitConnectOptions } from '../../utils/connect';
+import type { CallTree } from '../../profile-logic/call-tree';
+import type { IndexIntoCallNodeTable } from '../../types/profile-derived';
+
+type StateProps = {|
+  +tree: CallTree,
+  +selectedNodeIndex: IndexIntoCallNodeTable | null,
+|};
+
+type SidebarDetailProps = {|
+  +label: string,
+  +children: React.Node,
+|};
+
+function SidebarDetail({ label, children }: SidebarDetailProps) {
+  return (
+    <React.Fragment>
+      <div className="sidebar-label">{label}:</div>
+      {children}
+    </React.Fragment>
+  );
+}
```

Comment:
Note: we should use a stateless component for `_markerDetail` in tooltips too: https://github.com/devtools-html/perf.html/blob/master/src/components/shared/MarkerTooltipContents.js#L37-L49